examples/async.py

```diff
@@ -89,9 +89,13 @@ def delete(self, id_):
         return False
 
     @gen.coroutine
-    def list_(self):
+    def list_(self, limit=0, page=0):
         return [Posts.ResourceObject(self, p) for p in self.data]
 
+    @gen.coroutine
+    def list_count_(self):
```

Comment:
I suggest naming it `list_count` or simply `count`, because there is no name conflict with built-in function, as there is with `list`.